BridgeSDK/BridgeAPI/BridgeAPITypes/SBBActivity.m

```diff
@@ -33,6 +33,9 @@ @implementation SBBActivity
 
 #pragma mark Abstract method overrides
 
-// Custom logic goes here.
+- (instancetype)copyWithIdentifier: (NSString *)identifier {
+    SBBActivity *copy = [self copy];
+    return copy;
```

Comment:
...should it be updating the guid on the copy here?

Reply:
No. I considered that but since this is a wrapper implementation of `RSDTaskInfo`, it's not really appropriate. That said... just realized that I am not sure how `identifier` is mapped... so... let me get back to you on that. ;)

Reply:
@Erin-Mounts 
Updated to copy the identifier to whichever reference pointer is non-nil. Thanks!